### PR TITLE
Users/lilixie/login accessibility

### PR DESCRIPTION
**Pull request by Goodyear2017**
Summary of the changes (in less than 80 characters):

* change the html markups
* change related css in less file

Addresses https://github.com/NuGet/NuGetGallery/issues/9478

### CODE REVIEW

**Comment by ryuyu** [comment]
Similar to the comment that @agr made at https://github.com/NuGet/NuGetGallery/pull/9484#discussion_r1179501879, can we get a compiled css checked in as well?

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 1]
```diff
@@ -42,15 +42,31 @@
...
                 @:<b>
-            }
+                }
             <span>@tab</span>
             @if (bold)
             {
-                @:</b>
-            }
+            @:</b>
+        }
```
Comment (agr) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml:50]: indentation for the closing brace does not match the opening one. The line above is not indented properly.

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 2]
```diff
@@ -42,15 +42,31 @@
...
         </a>
     </li>
 }
+@helper DisplaySignInButton(string tab, string url, bool bold = false, string title = null) //Change the list item for accessibility
+{
+    <div class="">
```
Comment (agr) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml:56]: Remove empty `class` attribute.

Comment (agr) [reply src/NuGetGallery/Views/Shared/Gallery/Header.cshtml:56]: Why `li` was replaced with `div`?

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 3]
```diff
@@ -42,15 +42,31 @@
...
+        <a role="button" aria-expanded="@(ViewBag.Tab == tab ? "true" : "false")" href="@url" title="@title">
+            @if (bold)
+            {
+                @:<b>
+                }
+            <span>@tab</span>
+            @if (bold)
+            {
+            @:</b>
+        }
```
Comment (agr) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml:66]: Broken indentation.

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 4]
```diff
@@ -42,15 +42,31 @@
...
             @if (bold)
             {
-                @:</b>
-            }
+            @:</b>
+        }
         </a>
     </li>
 }
+@helper DisplaySignInButton(string tab, string url, bool bold = false, string title = null) //Change the list item for accessibility
```
Comment (agr) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml:54]: This helper looks like a copy-paste of `DisplayNavigationItem` with a single change (`li` -> `div`). Could we have it less copy-pasty? I'd suggest moving the internal part (`<a>...</a>`) into the helper of its own and calling into it from both `DisplayNavigationItem` and `DisplaySignInButton`.

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 5]
```diff
@@ -92,16 +108,17 @@
                     </ul>
                     @if (ShowAuthInHeader)
                     {
-                        <ul class="nav navbar-nav navbar-right" role="tablist">
+                        <div class="nav navbar-nav navbar-right">
```
Comment (agr) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml:111]: I'd move that line inside that `if ... else ...` one line below, keep it `<ul... role="tablist">` for the unauthenticated case (ant then there would be no need to have `DisplaySignInButton`, `DisplayNavigationItem` would be fine), then replace it with `div` for the authenticated and keep the rest of the markup changes.

**Comment by mariaghiondea** [comment]
@Goodyear2017 are you still interested in merging this PR? There are outstanding comments on it.

**Comment by mariaghiondea** [comment]
please reopen this and address the comments if it's still need it

**Pull request closed without merging**